docs/standard/library-guidance/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Open-source library guidance
+description: Best practice recommendations for developers to create high quality .NET libraries.
+author: jamesnk
+ms.author: James.NewtonKing
+ms.date: 09/20/2018
+---
+# Open-source library guidance
+
+This guidance provides recommendations for developers to create high-quality .NET libraries. This documentation focuses on the *what* and the *why* when building a .NET library, not the *how*.
+
+Aspects of high-quality open-source .NET libraries:
+
+> [!div class="checklist"]
+> * **Inclusive** - Good .NET libraries strive to support many platforms and applications.
```

Comment:
Also, include many different kinds of People, by including a Code Of Conduct!

Reply:
Hmm, while I like the spirit these guidelines are really focused on how to build great .NET libraries. I've told @JamesNK that I think we should be de-emphasizing OSS in these, because most equally applies to closed source too. I don't think we should turn these guidelines into a general "how build a great .NET OSS project". However, I do think this would be a great topic to write as well.

Reply:
Yeah that's true. This is foremost a guide about building good .NET libraries, not running a good OSS .NET projects.